src/test/java/teammates/test/cases/storage/FeedbackSessionsDbTest.java

```diff
@@ -211,6 +211,33 @@ private void testGetSoftDeletedFeedbackSessionsForCourse() {
         assertTrue(fsDb.getSoftDeletedFeedbackSessionsForCourse("idOfCourseNoEvals").isEmpty());
     }
 
+    @Test
+    public void testSoftDeleteFeedbackSession() throws Exception {
+        FeedbackSessionAttributes fs = getNewFeedbackSession();
+        fsDb.createEntity(fs);
+
+        ______TS("Success: soft delete an existing feedback session");
+        fsDb.softDeleteFeedbackSession(fs.getFeedbackSessionName(), fs.getCourseId());
+
+        assertNull(fsDb.getFeedbackSession(fs.getFeedbackSessionName(), fs.getCourseId()));
```

Comment:
`fsDb.getFeedbackSession(fs.getCourseId(), fs.getFeedbackSessionName())`?

Reply:
fixed, sorry for that.
Ready for review.